backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointResourceTest.java

```diff
@@ -697,6 +703,20 @@ void addOpenBridgeEndpoint() {
 
             ep.getProperties(CamelProperties.class).getExtras().put("channel", "#updated");
             ep.getProperties(CamelProperties.class).setUrl("https://updated.com");
+
+            /*
+             * Update the endpoint's status to make it seem as it its processor
+             * was ready to be updated. The transaction is required for
+             * Hibernate to run the update query.
+             */
+            final Transaction transaction = this.databaseSession.beginTransaction();
+            this.entityManager
+                .createQuery("UPDATE Endpoint SET status = :status WHERE id = :id")
+                .setParameter("status", EndpointStatus.READY)
+                .setParameter("id", UUID.fromString(id))
+                .executeUpdate();
+            transaction.commit();
```

Comment:
It would be a bit simpler to run that query from a method (which must NOT be `private`) annotated with `@Transactional`. It would rely on `EntityManager` instead of requiring a `Session` here. What you did works though so I don't mind approving this version.

Reply:
I did it and it works perfectly, thank you!